src/renderer/views/Channel/Channel.vue

```diff
@@ -151,10 +160,29 @@
             :channel-name="channel.author || channel.channelName"
             :channel-id="channel.channelId"
             :channel-thumbnail="channel.authorThumbnails[channel.authorThumbnails.length - 1].url"
+            role="link"
```

Comment:
@absidue i believe this is why link is checked in the ft-channel-bubble component